pkg/webui/lib/components/with-locale.js

```diff
@@ -92,7 +102,7 @@ export default class UserLocale extends React.PureComponent {
     }
   }
 
-  load(language) {
+  async load(language) {
     let locale = navigator.language || navigator.browserLanguage || 'en-US'
```

Comment:
```suggestion
    let locale = navigator.language || navigator.browserLanguage || defaultLanguage
```

Reply:
I think `defaultLocale` is more appropriate here